src/DDTrace/Integrations/Predis.php

```diff
@@ -20,6 +20,18 @@ class Predis
      */
     private static $connections = [];
 
+    public static function unload()
```

Comment:
This methods seems to be not called, did you maybe intend to call in the integration test's `tearDownAfterClass`?

Reply:
Ups sorry, this is a leftover from my work on unloading extensions from the data structures. Removing